pkg/prometheus/converter.go

```diff
@@ -0,0 +1,34 @@
+package prometheus
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/alphagov/paas-metrics/pkg/metric"
+)
+
+// Converter is responsible for generating Prometheus friendly metric entity in
+// a form of slice of bytes.
+func Converter(metrics []*metric.Metric) ([]byte, error) {
+	b := []string{}
+
+	for _, m := range metrics {
+		b = append(b, fmt.Sprintf("%s%s %g", m.Key, composeMeta(m), m.Value))
+	}
+
+	return []byte(strings.Join(b, "\n")), nil
+}
+
+func composeMeta(m *metric.Metric) string {
+	if m.Meta != nil {
```

Comment:
looks a bit unnatural to return the happy path inside the `if` block ... could this be refactored to do:

```
if m.Meta == nil {
return ""
}
```